unicsv.cc

```diff
@@ -366,7 +366,7 @@ unicsv_parse_gc_id(const QString& str)
   }
 
   long long res = 0;
-  for (auto c : s) {
+  for (auto c : qAsConst(s)) {
```

Comment:
CI rocks!

Reply:
Yes, that's nice indeed. Especially since I do not have the latest version of all the checker tools everywhere.